src/store/getters/state-getters.ts

```diff
@@ -387,6 +393,14 @@ export const hasContactInfoChange = (state: StateIF): boolean => {
     (businessContact.extension !== getSnapshotContact(state).extension)
 }
 
+/** Check for changes between current contact and original contact. */
+export const hasShareStructureChanges = (state: StateIF): boolean => {
+  const originalShareClasses = cloneDeep(state.stateModel.originalSnapshot?.shareStructure.shareClasses)
+  const currentShareClasses = cloneDeep(state.stateModel.shareStructureStep.shareClasses)
```

Comment:
Why do you need to clone it to compare it?

Reply:
I don't actually, being a bit over cautious here. 
The root issue of some issues in the past was references with in share structure overriding something i didn't want to override.. 
perhaps it made me over cautious.. i can remove :)

Reply:
I think these can be removed, since I believe `isEqual()` only compares the objects without changing them.

Reply:
Absolutely. I've been condition to fear object comparisons...

Reply:
Removed!